R/cut_dataToWaterYear.R

```diff
@@ -0,0 +1,48 @@
+#' Cuts the discharge time series to full water years
+#' @param x A dataframe containing a vector of date values in the first 
+#' column and vector of numeric flow values in the second column.
+#' @param wyMonth The month of the year in which the water year starts. The water year begins on the first day of wyMonth.
+#' @return data.frame in original structure, but cut to full water years
+#' @details To ensure the input data is valid for use with other 
+#' EflowStats functions. 
+#' #' \enumerate{
+#'   \item First column must be of class `Date`.
+#'   \item Second must be of class `numeric`.
+#'   \item `wyMonth`input must be of class `integer`. 
+#' }
+#' @examples
+#' x <- sampleData[, c('date', 'discharge')]
+#' cut_dataToWaterYear(x,10L)
+#' @export
+
+
+
+cut_dataToWaterYear <- function(x,wyMonth=10L) {
```

Comment:
Good idea, yes. I changed the function so that it returns the same columns and column names as of the 'x' passed to it. For information on how the function is used, see last answer below.